WMCDuplicateRemover/Code/Wrappers/MicrosoftEventLogWrapper.cs

```diff
@@ -36,8 +34,8 @@ public override bool FoundEventForRecording(String seriesName, String episodeNam
             //These are exceptions since I had to go and delete off these because WMC screwed up its DRM and I wasn't actually
             //finished watching these
             //TODO: Add the ability to add exceptions to this program without hard coding here
```

Comment:
The only way I can think to approach adding the ability to add exceptions like that would be to just reference a list somewhere.  Is that what you're planning on doing?